tox.ini

```diff
@@ -1,31 +1,30 @@
 [tox]
 distshare = {homedir}/.tox/distshare
-envlist = py27-pytest{43,44,45,46},
-          py38-pytest{43,44,45,46,50,51,52,53,54,60,latest},
+envlist = py27-pytest46,
+          py38-pytest{46,54,60,61,62,latest,master},
```

Comment:
I would prefer to keep on testing all the minor versions of pytest that we claim to support (5.0, 5.1, 5.2, 5.3).